### DISCUSSION TITLE
Lazy even if outside viewport

### DISCUSSION THREAD

**Discussion by yassinetrbls** [livewire/livewire#6153] [1. Help] [answered]
Hello

How to use lazy to load components even if not in viewport ?

Thanks

**Comment by ducopro** [comment]
By not using lazy at all......

Op wo 2 aug. 2023 11:10 schreef yassinetrbls ***@***.***>:

**Reply by yassinetrbls** [reply]
Not using lazy makes livewire loading with the page. I want it async :)

**Comment by nicolagianelli** [comment]
It's missing this feature now.
I think it can be useful to use a component in lazy mode if it's slow to load the page early and let it loading outside the viewport so when the user scroll will find it already loaded.

@yassinetrbls You should propose this as a feature, a lazy mode with viewport loading mode or with page load like:
lazy.viewport and lazy without it for the classic lazy mode that loads it when the page is fully loaded
and let's see how many people need this feature..for me it's good to have and has more sense to be loaded when the page loads by default and the viewport as option..

**Reply by nicolagianelli** [reply, marked as answer]
@yassinetrbls I created a pull request with this feature implemented as "defer",
If Caleb will find it useful it could be merged: #6171
